app/javascript/components/markdown-editor.js

```diff
@@ -38,32 +37,38 @@ MarkdownEditor.prototype.handlePreviewButton = function (event) {
   }
 
   var $preview = this.$preview
+  var $module = this.$module
   var text = this.$input.value
 
   // Mirror textarea's height
   $preview.style.height = this.$input.offsetHeight + 'px'
 
-  if (text) {
-    // Render markdown
-    marked(
-      text,
-      function (err, content) {
-        if (err) {
-          $preview.innerHTML = 'Error previewing content'
-          throw err
-        } else {
-          $preview.innerHTML = content
-        }
-      }
-    )
-    // Open preview links in a new tab
-    this.setTargetBlank(this.$preview)
-  } else {
+  if (!text) {
     $preview.innerHTML = 'Nothing to preview'
+    return
   }
+
+  var path = $module.getAttribute('data-govspeak-path')
+  var url = new URL(document.location.origin + path)
```

Comment:
`URL` and `searchParams` are not supported in any version of Internet Explorer

Reply:
I'll go for Polyfill - die die XMLHttpRequest!